composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "Generate Gherkin steps from Codeception Modules.",
   "type": "library",
   "require": {
-    "codeception/codeception": "^2.2"
+    "codeception/codeception": "^3.0"
```

Comment:
I use this in projects using `2.*`; what about `"codeception/codeception": ">=2.2"?